app/src/system/java/org/mozilla/tv/firefox/ext/EngineView.kt

```diff
@@ -62,6 +65,14 @@ fun EngineView.pauseAllVideoPlaybacks() {
     evalJS("document.querySelectorAll('video').forEach(v => v.pause());")
 }
 
+fun EngineView.domElementCache() {
+    evalJS(CACHE_JS)
+}
+
+fun EngineView.domElementRestore() {
+    evalJS("if ($CACHE_VAR) $CACHE_VAR.focus();")
+}
```

Comment:
Nit: I found these names a little bit confusing, would personally prefer `cacheDomElement` and `restoreDomElement`.